### PR TITLE
Implement twig hooks for account layout

### PR DESCRIPTION
**Pull request by bartek-sek**
image

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-    {% set description = 'sylius.ui.add_address'|trans %}
-    {{ parent() }}
-{% endblock %}
-
-{% block account_content %}
-    {% hook 'sylius_shop.account.address_book.create' with { form } %}
+{% block content %}
+    <div class="container mb-auto">
+        <div class="row my-4">
+            {% hook 'layout' with { _prefixes: ['sylius_shop.account.address_book.create', 'sylius_shop.shared.account'], form } %}
```
Comment (Wojdylak) [inline src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig]: `['sylius_shop.account.address_book.index', 'sylius_shop.shared.account']`
This should be done by setting the `prefixes` variable, similar to how it's done in `src/Sylius/Bundle/ShopBundle/templates/account/order/index.html.twig`. This way manage the 3 hooks (with  `sylius_shop.base.layout`).

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml [hunk 2]
```diff
@@ -1,20 +1,30 @@
 twig_hooks:
     hooks:
-        'sylius_shop.account.address_book.create':
+        'sylius_shop.account.address_book.create.layout':
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml]: I'm not convinced to the `layout` name 🤔 
```suggestion
        'sylius_shop.account.address_book.create.content':
```

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
+            breadcrumbs:
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml:4]: Shouldn't breadcrumb be a part of header 🤔

File: src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-    {{ parent() }}
-{% endblock %}
+{% block title %}{{ 'sylius.ui.add_address'|trans }} | {{ parent() }}{% endblock %}
 
-{% block account_content %}
-    {% hook 'sylius_shop.account.address_book.create' with { form } %}
+{% block content %}
+    <div class="container mb-auto">
+        <div class="row my-4">
+            {% hook 'layout' with { _prefixes: prefixes, form } %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig]:
```suggestion
            {% hook 'content' with { _prefixes: prefixes, form } %}
```
the same applies to other pages

File: src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig [hunk 5]
```diff
@@ -1,13 +1,13 @@
-{% extends '@SyliusShop/account/address_book/layout.html.twig' %}
+{% extends '@SyliusShop/shared/layout/base.html.twig' %}
 
-{% block title %}{{ 'sylius.ui.add_address'|trans }} | {{ parent() }}{% endblock %}
+{% set prefixes = ['sylius_shop.account.address_book.create', 'sylius_shop.shared.account'] %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig]: I wonder if we shouldn't change `shared` keyword to `common` in entire shop to be more consistent with admin 🤔 
```suggestion
{% set prefixes = ['sylius_shop.account.address_book.create', 'sylius_shop.common.account'] %}
```

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml (hunk 3 above)
Comment (bartek-sek) [reply src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml:4]: Header is part of the content and in that case breadcrumbs would be in a wrong position

**Review by TheMilek — DISMISSED** [review]

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml (hunk 3 above)
Comment (TheMilek) [reply src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml:4]: maybe we should then just add the header hook to be consistent with other places 🤔

**Review by GSadee — DISMISSED** [review]
But the rebase is needed

File: src/Sylius/Bundle/ShopBundle/templates/shared/account/layout/menu.html.twig [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div class="col-12 col-md-3">
+    {{ knp_menu_render('sylius.shop.account', {'template': '@SyliusShop/account/menu/content.html.twig'}) }}
```
Comment (Wojdylak) [inline src/Sylius/Bundle/ShopBundle/templates/shared/account/layout/menu.html.twig]:
```suggestion
    {{ knp_menu_render('sylius_shop.account', {'template': '@SyliusShop/account/menu/content.html.twig'}) }}
```

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/common/create.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_shop.account.common.create':
+            breadcrumbs:
+                template: '@SyliusShop/shared/account/layout/breadcrumbs.html.twig'
```
Comment (Wojdylak) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/common/create.yaml]:
```suggestion
                template: '@SyliusShop/account/common/layout/breadcrumbs.html.twig'
```
All hooks.

**Review by Wojdylak — APPROVED** [review]

**Pull request merged**